libs/community/langchain_community/document_loaders/hugging_face_model.py

```diff
@@ -0,0 +1,91 @@
+import requests
+from typing import Iterator, List, Optional
+
+from langchain_core.documents import Document
+from langchain_community.document_loaders.base import BaseLoader
+
+class HuggingFaceModelLoader(BaseLoader):
+    """Load model information from `Hugging Face Hub`, including README content."""
+
+    BASE_URL = "https://huggingface.co/api/models"
+    README_BASE_URL = "https://huggingface.co/{model_id}/raw/main/README.md"
+
+    def __init__(
+        self,
+        search: Optional[str] = None,
```

Comment:
Please use `*` oeprator prior to all named arguments

```suggestion
        *,
        search: Optional[str] = None,
```